runtime/doc/lsp.txt

```diff
@@ -1469,6 +1462,58 @@ save({lenses}, {bufnr}, {client_id})                 *vim.lsp.codelens.save()*
       • {client_id}  (integer)
 
 
+==============================================================================
+Lua module: vim.lsp.inlay_hint                                *lsp-inlay_hint*
+
+get({opts})                                         *vim.lsp.inlay_hint.get()*
+    Get the list of inlay hints, (optionally) restricted by buffer, client, or
+    range. This method does not make an LSP request, but just queries locally
+    stored inlay hints. The returned inlay hints are unresolved.
+
+    Example usage: >lua
+        local hint = vim.lsp.inlay_hint.get({ bufnr = bufnr })[1]
+        resolved_hint = client.request(methods.inlayHint_resolve, hint.inlay_hint, callback, bufnr)
```

Comment:
I think we should use `'inlayHint/resolve'` here for consistency with other examples.

I prefer using `vim.lsp.protocol.Methods` in documentation examples btw, but see https://github.com/neovim/neovim/pull/24977 on why I didn't end up changing it.

Reply:
thx for context, changed